sdk/src/instruction_accounts.rs

```diff
@@ -0,0 +1,202 @@
+use std::collections::HashMap;
+
+use solana_program::{instruction::AccountMeta, pubkey::Pubkey};
+
+use crate::{
+    PROGRAM_ID_ACCOUNT_COMPRESSION, PROGRAM_ID_LIGHT_SYSTEM, PROGRAM_ID_NOOP, PROGRAM_ID_SYSTEM,
+};
+
+/// Collection of Light Protocol system accounts which are sent to the program.
+pub struct LightInstructionAccounts {
```

Comment:
we could rename to `LightInstructionAccountsBuilder` and `LightSystemAccounts` to `LightInstructionAccounts` or the other way around to keep naming intuitive for dev on and offchain.
wdyt @vadorovsky ?
cc @SwenSchaeferjohann .